docs/lab-3/README.md

```diff
@@ -455,5 +456,19 @@ Start by just identifying the outlier, then try to get the model to tell you bot
 <img src="../images/lab3/3.10-answer.png" alt="Answer" />
 </details>
 
+## 11. RAG Questions
+Additional questions for the example RAG issue:?
```

Comment:
Maybe this would be better posed like below?

```suggestion
**What additional questions can be included for the example RAG issue?**
```

Reply:
sounds good, thanks Rafael